src/main/java/de/tum/in/www1/artemis/repository/SubmissionPolicyRepository.java

```diff
@@ -11,4 +12,8 @@
 @Repository
 public interface SubmissionPolicyRepository extends JpaRepository<SubmissionPolicy, Long> {
 
+    @Query("""
+            SELECT sp FROM SubmissionPolicy sp
+            WHERE sp.programmingExercise.id = :#{#exerciseId}""")
```

Comment:
Why do we need a Query here? This should also work just by the method name.
```suggestion
```

Reply:
Failed to create query for method public abstract de.tum.in.www1.artemis.domain.submissionpolicy.SubmissionPolicy de.tum.in.www1.artemis.repository.SubmissionPolicyRepository.findByExerciseId(java.lang.Long)! No property 'exerciseId' found for type 'SubmissionPolicy'